router/auth.js

```diff
@@ -5,17 +5,19 @@ const router = express.Router()
 
 router.post('/register',async(req,res)=>{
 
-   const{name,email,phoneno,password}=req.body;
-   if(!name||!email||!phoneno||!password){
+   const{name,email,createpassword,confirmpassword}=req.body;
```

Comment:
We should remove confirmpassword and change createpassword to only password.
```suggestion
   const { name, email, password } = req.body;
```